packages/toolkit/package.json

```diff
@@ -31,13 +31,36 @@
   ],
   "author": "Aragon Association <legal@aragon.org>",
   "license": "GPL-3.0-or-later",
+  "dependencies": {
+    "@aragon/apm": "^3.1.2-beta.2",
```

Comment:
Modified version of `@aragon/apm` that use `@aragon/abis` instead of `@aragon/os`.